tests/main/system-usernames-microk8s/microk8s/meta/snap.yaml

```diff
@@ -0,0 +1,9 @@
+name: microk8s
```

Comment:
this snap.yaml and the test-microk8s-username snap both have identical snap.yaml and bin/sh, we can instead use sed to generate a snap.yaml for each one to replace the name here. See where I have done this for example: https://github.com/snapcore/snapd/blob/master/tests/core/netplan/task.yaml